src/main/java/Deadlines.java

```diff
@@ -0,0 +1,13 @@
+public class Deadlines extends Task {
```

Comment:
Is it better to use a singular instead of plural noun for the class name? (same goes for ToDos and Events, since Task is in singular)